ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/operations/SlashableAttestation.java

```diff
@@ -55,9 +52,7 @@ public static SlashableAttestation fromBytes(Bytes bytes) {
   public Bytes toBytes() {
     return SSZ.encode(
         writer -> {
-          writer.writeULongIntList(
-              64,
-              validator_indices.stream().map(UnsignedLong::longValue).collect(Collectors.toList()));
+          writer.writeLongIntList(64, validator_indices);
```

Comment:
This is a little bit easier and takes that magic number 64 out of the code.

```suggestion
          writer.writeInt64List(validator_indices);
```